apps/vault/test/vault.js

```diff
@@ -128,9 +134,90 @@ contract('Vault app', (accounts) => {
   })
 
   context('using Vault behind proxy', async () => {
-    // TODO(BLOCKED): Waiting for aragonOS#281
-    it('forwards funds')
-    it('fails when attempting to recover assets out of the vault')
+    let kernel, token, vault, vaultBase, vaultId
+
+    beforeEach(async () => {
+      const permissionsRoot = accounts[0]
+      token = await SimpleERC20.new()
+
+      const kernelBase = await getContract('Kernel').new()
+      const aclBase = await getContract('ACL').new()
+      const factory = await DAOFactory.new(kernelBase.address, aclBase.address, NULL_ADDRESS)
+
+      const receipt = await factory.newDAO(permissionsRoot)
+      const kernelAddress = getEvent(receipt, 'DeployDAO', 'dao')
+
+      kernel = Kernel.at(kernelAddress)
+      const acl = ACL.at(await kernel.acl())
+
+      const r = await kernel.APP_MANAGER_ROLE()
+      await acl.createPermission(permissionsRoot, kernel.address, r, permissionsRoot)
+
+      // Install the vault app to the kernel
+      vaultBase = await Vault.new()
+
+      vaultId = hash('vault.aragonpm.test')
+      const APP_BASE_NAMESPACE = await kernelBase.APP_BASES_NAMESPACE()
+
+      const vaultReceipt = await kernel.newAppInstance(vaultId, vaultBase.address)
+      const vaultProxyAddress = getEvent(vaultReceipt, 'NewAppProxy', 'proxy')
+      vault = Vault.at(vaultProxyAddress)
+    })
+
+    it('forwards funds', async () => {
```

Comment:
@izqui I wasn't quite sure what this test that you had scaffolded from before was supposed to do.

Reply:
Trying to come up with an excuse but I don't remember at all tbf :)